webpack.config.ts

```diff
@@ -52,7 +51,7 @@ module.exports = {
         {
           'process.env.NODE_ENV': process.env.NODE_ENV,
         },
-        v => JSON.stringify(v),
+        (v: any) => JSON.stringify(v),
       ),
     ),
     ...ifProd([new BabelMinifyPlugin()]),
```

Comment:
webpack config in TS 🤔 Cool!

Reply:
This is yet another workaround to force Webpack to compile the config file to JS compatible with Node 6 :sweat:

Reply:
Why is AWS Lambda still on Node 6? Do they have a good reason for staying on that or are they just slow to upgrade? If they have a good reason, maybe we should adopt Node 6 across the board to simplify our stack?

Reply:
No good reason I guess. The current LTS version is 8.9 and the next LTS will be version 10 and it will be released next month per [the release schedule](https://github.com/nodejs/Release). Also, according to many articles, there's a huge performance difference between 6 and 8.9.

That said, I think we should compile the helpers in `hollowverse/common` with `@babe/preset-env` to make it work in Node 6. This should help remove some of the workarounds we use here.